data/index.php

```diff
@@ -279,7 +289,15 @@ function selectSet(id) {
 				</select><br/>
 				<input type="text" name="ids" id="idlist" placeholder="2,5,19-23" style="width:330px; margin-top:5px;"/>
 			</fieldset>
-			
+			<fieldset id="limit">
+				<legend>Limit results (0=all)</legend>
+				<input type="text" name="limit" id="limit" placeholder="0" style="width:90%"/>
+			</fieldset>
+			<fieldset id="sort">
+				<legend>Sort order</legend>
+				<input type="radio" name="sort_order" value='ASC' checked>Oldest first<br/>
+				<input type="radio" name="sort_order" value='DESC'>Newest first
```

Comment:
I'm missing a `</input>` here.

Reply:
That was what I thought when I wrote the code the first time, and then I saw Firefox removing the tags, so I went to https://www.w3schools.com/tags/att_input_type_radio.asp and found that radio buttons should not have a </input>.

Reply:
Ah, good point. So the text is not actually *in* the input tag, just after it. Looks good, then :-)